src/cc_catalog_airflow/dags/provider_api_scripts/wikimedia_commons.py

```diff
@@ -103,25 +103,24 @@ def _get_image_batch(
         end_ts,
         continue_token=continue_token
```

Comment:
It may be useful to include a bit of inline documentation in this function (the rest of the file is also a little scant on docstrings, but that's outside of the scope of this review). I would recommend a docstring explaining the signature of the function variables (particularly `start_ts` and `end_ts`; presumably the "ts" means "time series", but I'm not 100% sure) as well as the continuation token.

Reply:
I think better than a docstring for an internal function would be to expand 'ts' to 'timestamp'.  Perhaps we should have a conversation about what our expectations are for docstrings on internal vs external functions.  I usually consider docstrings to be something that should help someone who doesn't have the code open in their editor at that moment.  Usually, if things aren't clear when someone *does* have the code open in their editor, I consider it a defect of variable or function naming.

Reply:
Additional note:  The 'main' function is the only public function of the file.  I'll add a docstring to that one giving a bit of info.